src/exercises-final/05.js

```diff
@@ -13,7 +13,6 @@ class Toggle extends React.Component {
   getStateAndHelpers() {
     return {
       on: this.state.on,
-      toggle: this.toggle,
```

Comment:
This is still useful in some situations, so I definitely suggest continuing to expose this. Remember that prop getters are just helpers and should generally not be the _only_ way to accomplish certain tasks :) Thanks anyway!